api/v1alpha1/worker.go

```diff
@@ -72,3 +75,45 @@ func (worker Worker) ErrorCondition(msg string) Condition {
 		LastSuccessfulTime: metav1.Now(),
 	}
 }
+
+func (worker Worker) BuildEmbedder() *Embedder {
+	return &Embedder{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: worker.Namespace,
+			Name:      worker.Name + "-worker",
```

Comment:
Not sure  about this `Name`. Maybe a random string?

Reply:
@nkwangleiGIT @Abirdcfly @0xff-dev any comments?

Reply:
<img width="595" alt="image" src="https://github.com/kubeagi/arcadia/assets/30621793/302bbacd-cd9d-432a-a7e2-da5e74ad13cc">